### DISCUSSION TITLE
Max width button

### DISCUSSION THREAD

**Discussion by marzung** [family/connectkit#267] [Q&A] [answered]
Is there an easy way to create a max width button for example:
![image](https://github.com/family/connectkit/assets/100881747/7fa8c24c-89f2-43a4-9191-8e17a50d38d0)

**Comment by lochie** [comment, marked as answer]
Hey @marzung, you can create a custom Connect Wallet button using our custom component setup.
https://docs.family.co/connectkit/connect-button
